[2022-09-08T11:44:35.795+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: second_dag_v4.python2 manual__2022-09-08T11:44:35.187774+00:00 [queued]>
[2022-09-08T11:44:35.801+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: second_dag_v4.python2 manual__2022-09-08T11:44:35.187774+00:00 [queued]>
[2022-09-08T11:44:35.801+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T11:44:35.802+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-08T11:44:35.803+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T11:44:35.811+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): python2> on 2022-09-08 11:44:35.187774+00:00
[2022-09-08T11:44:35.814+0000] {standard_task_runner.py:52} INFO - Started process 3015 to run task
[2022-09-08T11:44:35.817+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'second_dag_v4', 'python2', 'manual__2022-09-08T11:44:35.187774+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpk184jv1f', '--error-file', '/tmp/tmpvrrhxj2s']
[2022-09-08T11:44:35.818+0000] {standard_task_runner.py:80} INFO - Job 52: Subtask python2
[2022-09-08T11:44:35.855+0000] {task_command.py:371} INFO - Running <TaskInstance: second_dag_v4.python2 manual__2022-09-08T11:44:35.187774+00:00 [running]> on host e458dbb0fb9f
[2022-09-08T11:44:35.894+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=second_dag_v4
AIRFLOW_CTX_TASK_ID=python2
AIRFLOW_CTX_EXECUTION_DATE=2022-09-08T11:44:35.187774+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-08T11:44:35.187774+00:00
[2022-09-08T11:44:35.895+0000] {python.py:173} INFO - Done. Returned value was: vua trai dat
[2022-09-08T11:44:36.060+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=second_dag_v4, task_id=python2, execution_date=20220908T114435, start_date=20220908T114435, end_date=20220908T114436
[2022-09-08T11:44:36.116+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-08T11:44:36.140+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
